### ISSUE TITLE
Unable to install via npm due to invalid component-cdn-uploader

### ISSUE THREAD

**Issue by Enngage** [auth0/lock#1703]
Hi,

Today I started getting following error when installing `"auth0-lock": "11.17.0"`:

```
npm WARN deprecated core-js@1.2.7: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
npm ERR! path git
npm ERR! code ENOENT
npm ERR! errno ENOENT
npm ERR! syscall spawn git
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/auth0/component-cdn-uploader.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

I tried installing `auth0-lock` dependency on https://stackblitz.com, but got same result: 

![asefase](https://user-images.githubusercontent.com/7137351/61567209-11571900-aa7f-11e9-9e19-b13342bd7cd8.png)

Can someone shed some light on this? 

Thanks!

**Comment by luisrudge** [comment]
Sorry! There was two errors here: first, this dependency should be a development dependency. The other one is that we should load if via npm and not git. I opened a PR to fix this and we'll have this fixed in a patch fix on Monday!

**Comment by luisrudge** [comment]
Fixed in 11.17.1
